README.yaml

```
@@ -39,6 +39,9 @@ related:
   - name: "terraform-null-label"
     description: "Terraform module designed to generate consistent names and tags for resources. Use terraform-null-label to implement a strict naming convention."
     url: "https://github.com/cloudposse/terraform-null-label"
+  - name: "terraform-aws-ssm-patch-management"
+    description: "This module was heavely inspired by @jparnaudeau module"
```

```suggestion
    description: "Terraform module to provision AWS SSM Patch Manager maintenance window tasks, targets, patch baseline, patch groups and an s3 bucket for storing patch task logs.
"
```

The attribution should not be in the description.
Add a separate section to README with the attribution